template.yaml

```diff
@@ -10,9 +10,9 @@ Metadata:
     Name: PublicationChannelRegister
     Description: Channel Registry for publications
     Author: Unit
-    SemanticVersion: 1.0.0
```

Comment:
We don't use SemanticVersion anymore?

Reply:
Moved to command line, but we could  use envsubst and keep it in template